slave-base/contrib/bin/run-jnlp-client

```diff
@@ -21,14 +21,12 @@ source /usr/local/bin/generate_container_user
 JVMPath32bit=`alternatives --display java | grep family | grep i386 | awk '{print $1}'`
 JVMPath64bit=`alternatives --display java | grep family | grep x86_64 | awk '{print $1}'`
 
-# set the java version used based on OPENSHIFT_JENKINS_JVM_ARCH
-if [ -z $OPENSHIFT_JENKINS_JVM_ARCH  ]; then
-    echo "Using 64 bit Java since OPENSHIFT_JENKINS_JVM_ARCH is not set"
-    alternatives --set java $JVMPath64bit
-elif [ "${OPENSHIFT_JENKINS_JVM_ARCH}" == "x86_64"  ]; then
+# If OPENSHIFT_JENKINS_JVM_ARCH is explicitly set, set the JVM architecture
+# used.  If not explicitly set, the architecture is decided below.
+if [[ "${OPENSHIFT_JENKINS_JVM_ARCH}" == "x86_64" ]]; then
     echo "64 bit Java explicitly set in OPENSHIFT_JENKINS_JVM_ARCH"
     alternatives --set java $JVMPath64bit
-else
+elif [[ "${OPENSHIFT_JENKINS_JVM_ARCH}" ]]; then
```

Comment:
I haven't noticed the duplicate logging in the master logs (but maybe I've just always missed it?), but does it make sense if the same fix planned in either this PR a separate PR for the master run script?  Or was something different that prevented duplicate logging from occurring there?

Reply:
https://github.com/openshift/jenkins/issues/423 ;-)
I'll keep a single PR for now but close that issue off when it goes in.